src/types/function.rs

```diff
@@ -74,7 +74,7 @@ impl PyCFunction {
         )
     }
 
-    /// Create a new built-in function without keywords.
+    /// Create a new built-in function without keywords (no arguments).
```

Comment:
This was confusing to me as it calls `&PyMethodDef::noargs()` below - this might be a quirk of the Python C-API which I'm not super familiar with.

Reply:
Good catch. How about making this even more explicit?

```suggestion
    /// Create a new built-in function which takes no arguments.
```

Reply:
That's good I just wasn't sure if "keywords" had some meaning in the C-API that might be helpful for others with that experience.